lib/Runtime/Library/InJavascript/Array_prototype.js

```diff
@@ -225,33 +225,78 @@
         }
     });
 
-    platform.registerChakraLibraryFunction("DefaultSortCompare", function(left, right) {
-        // only have to handle the less than case due to the logic of the sort
-        return `${left}` < `${right}` ? -1 : 0;
+    platform.registerChakraLibraryFunction("InsertionSort", function(array, length, compareFn) {
```

Comment:
Moved insertion sort into a helper function as I now call it in two places - this may be a bad change.